docs/_docs/collections.md

```diff
@@ -89,18 +89,59 @@ choice and written out to `<dest>/my_collection/some_subdir/some_doc.html`.
 
 ## Configuring permalinks for collections {#permalinks}
 
-You can customize the [Permalinks](../permalinks/) for your collection's documents by setting `permalink` property in the collection's configuration as follows:
+You can specify a pattern for the URLs where your Collection documents will reside with the [`permalink` property](../permalinks/):
 
 ```yaml
 collections:
-  my_collection:
-    output: true
-    permalink: /awesome/:path/:title.:output_ext
+  categories:
+    output: true                  # without this flag, this Collection's documents will not be generated at build time
+    permalink: /category/:name    # custom permalink definition
 ```
 
-In this example, the collection documents will the have the URL of `awesome` followed by the path to the document and its file extension.
+### Examples
+
+For a collection named `apidocs` with the following source file structure,
+
+```
+├── _apidocs
+│   └── archive
+│       └── deprecated.md
```

Comment:
Ha, I was worried that might raise an eyebrow. 

I figured that a generic-but-realistic use case would make for a more descriptive example than `apidocs/mydocs/doc1.md`; _i.e.,_ that it would be easier to imagine why you might choose to set your permalinks up as, say, `doc/deprecated.html` than `awesome/doc1.html`. But I also wrote it in a hurry, and I recognize that maybe `apidocs/archive/deprecated.md` is not as realistic as it could be.

If it creates more confusion than it resolves, I'm happy to change it back to `apidocs/mydocs/doc1.md` and resubmit the PR.